GentApp/Views/HomePage.xaml

```diff
@@ -14,7 +14,6 @@
 	<interactivity:Interaction.Behaviors>
 		<core:EventTriggerBehavior EventName="Loaded">
 			<core:InvokeCommandAction Command="{Binding Path=LoadBranchesCommand}" />
-			<core:InvokeCommandAction Command="{Binding Path=LoadSubscriptionsCommand}" />
```

Comment:
wrm weg??? is toch nodig voor branchdetailspage de subscribe/unsubscribe button te setten

Reply:
kga verkeerd gepulled hebben ofzo kheb da nie weggedaan